### PR TITLE
Make "Kubernetes Resource Model (KRM)" more prominent

### PR DESCRIPTION
**Pull request by bgrant0607**
"KRM" is the term we have been using in practice. It was buried and/or implicit in this document. Making it more prominent and explicit to make it easier to find.

### CODE REVIEW

**Comment by bgrant0607** [comment]
Hmm. It's not obvious from https://github.com/kubernetes/test-infra/blob/master/prow/plugins/blockade/blockade.go how to remove the blocked-paths label. In the past, I probably just did that manually.

The label was created long ago:
https://groups.google.com/g/kubernetes-sig-contribex/c/yEO6Sl7psUY/m/xNR2k-X3CwAJ

Are no updates possible now?

File: contributors/design-proposals/architecture/resource-management.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ At the center of the Kubernetes control plane is the [apiserver](https://kuberne
...
...th the growth in functionality over the past four years, the number of built-in APIs grown by more than an order of magnitude. Moreover, Kubernetes now supports multiple API extension mechanisms that are not only used to add new functionality to Kubernetes itself, but provide frameworks for constructing an ecosystem of components, such as [Operators](https://coreos.com/operators/), for managing applications, platforms, infrastructure, and other things beyond the scope of Kubernetes itself. In addition to providing an overview of the common behaviors of built-in Kubernetes API resources, this document attempts to explain the assumptions, expectations, principles, conventions, and goals of the **Kubernetes Resource Model (KRM)** so as to foster consistency and interoperability within that ecosystem as the uses of its API mechanisms and patterns expand. Any API using the same mechanisms and patterns will automatically work with any libraries and tools (e.g., CLIs, UIs, configuration, deployment, workflow) that have already integrated support for the model, which means that integrating support for N APIs implemented using the model in M tools is merely O(M) work rather than O(NM) work.
```
Comment (krmayankk) [inline contributors/design-proposals/architecture/resource-management.md:11]: nit: `APIs have grown by more than`

**Comment by mrbobbytables** [comment]
@bgrant0607 blockade [was configured to prevent changes to design proposals earlier this year](https://github.com/kubernetes/test-infra/pull/17137) as that process has been superseded by KEPs. Folks kept opening PRs to add new design proposals instead of following the KEP process >_<

We do have an outstanding issue to [Separating proposals and architecture related items](https://github.com/kubernetes/community/issues/2565).  It might be time to do a quick audit and move relevant docs to our [developer documentation](https://github.com/kubernetes/community/tree/master/contributors/devel) and then move the rest over to the [enhancements repo](https://github.com/kubernetes/enhancements).

**Comment by dims** [comment]
@kubernetes/community-admins let's please merge this by hand.

**Comment by mrbobbytables** [comment]
> @kubernetes/community-admins let's please merge this by hand.

done 👍

**Comment by dims** [comment]
thanks @mrbobbytables

**Comment by bgrant0607** [comment]
Thanks!

Thanks for the pointers. Makes sense. Of course, I missed the github notifications and didn't see it announced on kubernetes-dev.

I think everything in the architecture directory could be moved to developer documentation. Other than maybe namespaces.md, these documents were retrospective rather than proposals. There are quite a few external links to some of the documents in their current location, however.

**Pull request merged**